src/com/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1094,48 +1094,70 @@
     (dissoc row-map :environment_id)
     row-map))
 
+(defn normalize-resource-event
```

Comment:
Maybe it's differing definitions or splitting hairs (or both?) but what I was intending by this function name was normalizing the values. Timestamps as strings can come in with different formats, old/new values can have maps that are sorted arbitrarily etc, but we need some baseline from whatever way it came in to something that is repeatable before we compute the hash.

Is there something that reveals that intention better than normalize?